theme/less/_navigation.less

```diff
@@ -93,13 +95,16 @@
 #main-navigation li.has-children > a:after
 {
   content: url('/images/arrow_bullet.png');
-  padding-left: 10px;
+  padding-left: 11px;
```

Comment:
Maybe it's right to do here `padding: 0 0 0 11px`.

Reply:
It will have same effect. Having `padding-left` seem to be fine.

Reply:
I see you still changed to your suggestion. Anyway it's not a big deal, though I feel more in favor of left padding, if there is no such thing as "padding-end"